src/chart/XYAxis.css

```diff
@@ -3,6 +3,10 @@
   font: 10px sans-serif;
 }
 
+.chart_XYAxis .tick > text {
+  font: 10px sans-serif;
+}
```

Comment:
I can remove it ... but i put it there to fix a bug i noticed i can't recall what the bug was 100% ... i believe  it was due to some css overwriting it cause the font size wasnt specific enough to the tick text